Source/Navigation/Web/src/index.html

```diff
@@ -1,20 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
     <base href="/" target="_parent">
 </head>
-<body>       
+
+<body>
     <header class="navigation-header">
         <figure class="logo">
             ${ require('./assets/images/ifrc.svg')}
         </figure>
         ${require('./Components/navigation.html') }
-    </header>   
-</body>
-</html>
 
+        <div class="login-status">
+
+            <!-- ko if: nationalSocieties().length > 1 -->
```

Comment:
Knockout?

Reply:
Yup.. 
The thing is that this particular page should be very small - ideally I'd not do any framework, but Knockout had the least impact of them all - sizewize.  In the interest of time I decided to do this.